pubspec.yaml

```diff
@@ -124,7 +124,7 @@ dependencies:
   # enough_html_editor: Compose email for Mobile & Tablet
   enough_html_editor:
     git:
-      url: https://github.com/dab246/enough_html_editor.git
```

Comment:
should move to Linagora repository?

Reply:
After finish all richtext function I will remove it

Reply:
okie #753